### ISSUE TITLE
Categories - Correct 3 dots menu

### ISSUE THREAD

**Issue by esteels** [ji-devs/ji-cloud#478]
The 3 dots menu on the Ji Cloud Categories page does not match the Zeplin sketches.

Zeplin sketch for typical 3 dots menu: https://www.loom.com/i/52a2bff72fe2412cae0e76e118b497bf

Ji Cloud for typical 3 dots menu: https://www.loom.com/i/a1febb52c4d040768fe43b493c9fc7a5

(1) Please add "Hide" to the 3 dots menu. 
(2) Remove "Move up" / "Move down" from the 3 dots menu and replace with arrows up and down? 

Zeplin sketch for "full categories" 3 dots menu: https://www.loom.com/i/f9201aadb6c34ff6996590237bb022e3 

Ji Cloud for "full categories" 3 dots menu: https://www.loom.com/i/042c41139f8747358e497f836120205d

(1) Please add "Hide" to the 3 dots menu. 
(2) Remove "Move up" / "Move down" from the 3 dots menu and replace with arrows up and down? 
(3) Add "Add a sibling" to the 3 dots menu.

**Comment by dani-ji** [comment]
@corinnewo and @dakom did we decide to remove hide?

**Comment by dakom** [comment]
we decided to hide hide

**Comment by dani-ji** [comment]
Ok, what about the add a sibling category?

**Comment by dakom** [comment]
> we decided to hide hide

No but seriously, I _vaguely_ remember taking it out because we weren't sure what it's supposed to do... if we add it back in then we need to brief LB on the implications for category listing etc.

**Comment by dakom** [comment]
> Ok, what about the add a sibling category?

We could add that in, not sure why it's missing

**Comment by dani-ji** [comment]
@corinnewo any input regarding siblings?

**Comment by corinnewo** [comment]
@dani-ji @dakom The HIDE functionality was so that we could hide certain categories from being displayed in the Advanced search. We would have put stickers in a category, but users would not need to search in those categories. I do think it is important to have. For example, categories for our prime clients should not be visible in an advanced search. Perhaps the word 'Hide' should be 'Hide from advanced search', or something more explicit.

Regarding siblings - I don't think it is essential. When I needed to add a sibling, I went up to the higher level, and added a child. Probably better to have less items in the menu.
